Formula/calicoctl.rb

```diff
@@ -0,0 +1,28 @@
+class Calicoctl < Formula
+  desc "Calico CLI tool"
+  homepage "https://www.projectcalico.org"
+  url "https://github.com/projectcalico/calicoctl.git",
+      :tag => "v1.6.1",
+      :revision => "1724e011ac0e608190d7d5512ab8028bcd18ae7b"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    pkg_name = "github.com/projectcalico/calicoctl"
+    build_dir = buildpath/"src/#{pkg_name}"
+    build_dir.install buildpath.children
+    cd build_dir do
+      system "glide", "install", "-strip-vendor"
+      system "make", "binary"
+      bin.install "dist/calicoctl-darwin-amd64" => "calicoctl"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/calicoctl", "--version"
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software.

Reply:
In all honesty this formula is alike to other programs already in master:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/peco.rb
https://github.com/Homebrew/homebrew-core/blob/master/Formula/mtr.rb

Not even sure how to test it further. This is a software that depends on a platform available to work.
Just wondering, have the requirements changed recently?

Reply:
yes.